lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -249,12 +244,11 @@ public SystemAccessControl create(Map<String, String> config)
 
         private void bindRules(Binder binder, FileBasedAccessControlConfig configuration)
         {
-            if (isRest(configuration)) {
+            if (FileBasedAccessControlUtils.isRest(configuration)) {
                 binder.bind(new TypeLiteral<AccessControlRulesProvider<FileBasedSystemAccessControlRules>>() {})
                         .to(RestFileBasedSystemAccessControlRulesProvider.class)
                         .in(Scopes.SINGLETON);
-                httpClientBinder(binder).bindHttpClient("s3SecurityMapping",
-                        ForFileBasedSystemAccessControlRules.class)
+                httpClientBinder(binder).bindHttpClient("systemAccessControlRulesClient", ForAccessControlRules.class)
```

Comment:
Please test that one cannot configure http client if local file is used. Also test that when http is used one can configure this client by passing http client related configuration (`io.airlift.http.client.HttpClientConfig`)

Reply:
Can you please elaborate a bit on your comment ? So for example how could I even try to "configure http client if local file is used" ? I was under the impression that, if I would set configuration properties for `HttpClientConfig` they would not have any effect since the code for binding the http-client is never reached (controlled by the `isRest` method which is simple enough that it does not need a unit test imho). 
Do you have any pointer on how I could retrieve the http-client instance once I created a `FileBasedSystemAccessControl` instance for testing via the factory to check if the `HttpClientConfig` has been applied?

Reply:
>  I was under the impression that [...] the code for binding the http-client is never reached

That my impression too. I just wanted to be sure it is the case it does not change in future. 

So if file based rules are used no should be able to set:
```
systemAccessControlRulesClient.http-client.trust-store-path=file.pem
```

Also notice that `systemAccessControlRulesClient` should be replaced with something more user friendly as it is exposed as configuration key.